v2/cf-nodejs-logging-support/src/lib/config/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+export interface MergedConfig {
```

Comment:
why is the merged config not a config object itself, is this differenciation neccessary in typescript?